server/controllers/recipe-controller.js

```diff
@@ -148,20 +152,30 @@ class RecipeController {
    * @param {Object} res Express response object
    * @returns {undefined}
    */
-  static getRecipes(req, res) {
-    Recipe.findAll(defaultRecipeDbFilter)
+  static async getRecipes(req, res) {
+    const { page } = req.query;
```

Comment:
This doesn't match the spec's requirements - this endpoint should accept an _offset_, not a _page_.